PeerioTechnologies/peerio-desktop
# feat: after downloading file, show it in folder instead of previewing it

Files are no longer previewed after downloading, instead the destination directory is opened with system file manager with the downloaded file selected.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### PR author

#### Relevant issue tracker links  
...
#### Linked PRs in other repos

https://github.com/PeerioTechnologies/peerio-icebear/pull/88

#### Manual testing instructions  

1. Download some file.
2. Make sure it opens in system file manager after downloading with the file selected instead of showing its preview.

#### Additional notes
... 

----
### Reviewer

* Localisation.
* Architectural decisions.
* Performance.
* Comments.
* Big feature tech design documentation.
* Readability.
* DRY.
* Can we extract something to SDK?
* Are there any bicycles invented that should be replaced with existing SDK
* Are there unit and e2e tests added?

----
### Repository owner
